denizen_scripts/survival/repo_link/systems/server_restarter.dsc

```diff
@@ -19,7 +19,7 @@ survival_restart_timer:
       - wait 4.5m
       - announce <&b> "The server will be performing a scheduled restart in 30 seconds. Please reach a safe location."
       - repeat 29:
-        - actionbar "<&c>The server will be restarting in <element[31].-[<[value]>]> seconds."
+        - actionbar "<&e>The server will be restarting in <&c><element[31].-[value]> <&e>seconds."
```

Comment:
This is invalid.

Reply:
![image](https://user-images.githubusercontent.com/25861695/98870051-88a87d00-2427-11eb-931f-3788b789421c.png)

Reply:
https://github.com/Adriftus-Studios/network-script-data/issues/217